static/src/stylesheets/module/content/_media.head.scss

```diff
@@ -22,3 +22,14 @@
 .media-body {
     position: relative;
 }
+
+.tonal--tone-media .content__headline {
+    .inline-icon {
+        margin-right: .1em;
+
+        svg {
+            width: .86em;
```

Comment:
It used to be height auto, but I changed it to % as it wasn't scaling correctly in Safari.